pages/linux/trap.md

```diff
@@ -0,0 +1,24 @@
+# trap
+
+> Automatically execute commands after receiving signals by processes or the operating system.
+> Can be used to perform cleanups for interruptions by the user or other actions.
+
+- List available signals to set traps for:
+
+`trap -l`
+
+- List active traps for the current shell:
+
+`trap -p`
+
+- Set a trap to execute commands when one or more signals are detected:
+
+`trap 'echo "Caught signal SIGHUP"' SIGHUP`
+
+- Remove active traps:
+
+`trap - SIGHUP SIGINT`
+
+- Set a trap to delete a file after the user has interrupted a download:
```

Comment:
We can now delete this since we have already mentioned the example above. That is what I meant - to use the example along with the description itself.